src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/ExpressionCompilerTests.cs

```diff
@@ -2335,9 +2335,9 @@ static void Main()
         }
 
         /// <summary>
-        /// Flow analysis should be run on the generated method.
+        /// Flow analysis is not run on the generated method.
```

Comment:
I would guess that the intention was to document the behavior.  I agree that the test should more explicitly explain that if it fails because flow analysis has started being applied, then the test should be updated or deleted.